### DISCUSSION TITLE
Markdown Preview Enhanced can not show plantuml diagram.

### DISCUSSION THREAD

**Discussion by sinojelly** [coder/code-server#2570] [Q&A]
It says "Error: write EPIPE”
![Screenshot_20210111_214850_vn vhn vsc](https://user-images.githubusercontent.com/520711/104331284-da55ac80-5529-11eb-8f5c-91574c708ec9.jpg)

**Comment by code-asher** [comment]
Do you have Java installed? It looks like the plantuml functionality
requires Java.

**Reply by sinojelly** [reply]
Thanks!  I tried to install java following this link: 
https://stackoverflow.com/questions/61720889/how-can-i-install-java-runtime-on-android-device

When I run this command:
apt-get install -y ./openjdk-9-jre-headless_9.2017.8.20-1_arm.deb

It says: Reading package lists  Error. please see the picture below.
Any help would be appreciated!
![Screenshot_20210116_125620_vn vhn vsc](https://user-images.githubusercontent.com/520711/104797714-30ad3e80-57fb-11eb-890b-ee5cc6f105d1.jpg)

**Reply by code-asher** [reply]
I'm not certain but maybe this will help: https://askubuntu.com/questions/30072/how-do-i-fix-a-problem-with-mergelist-or-status-file-could-not-be-parsed-err#30199
